src/Ninja/G1CP/Content/Fixes/Session/fix197_TemplarVIPDialog.d

```diff
@@ -0,0 +1,43 @@
+/*
+ * #197 Dialog lines with ambient templar interchanged pt. 2
+ */
+func int G1CP_197_TemplarVIPDialog() {
```

Comment:
I suggest the function to be `G1CP_197_TemplarCampDialog`. I had named the function of #142 "TemplarVIPDialog" because the question in the dialog regards the VIPs (important people) of the camp.